### PR TITLE
Update Midwood Isle

### PR DESCRIPTION
**Pull request by IllusiveMan196**
Main mod was cleaned by author couple versions ago. Changed data.
Added secondary ship location and dynamic light control by Tarlazo (MILightsControl).

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -15236,11 +15236,19 @@ plugins:
 
   - name: 'Midwood Isle.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/28120/' ]
+    clean:
+      - crc: 0x902458B5
+        util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
```
Comment (sibir-ine) [inline masterlist.yaml]: `util` strings for cleaning data mapped to a `clean` key don't get displayed as Markdown, nor would links be reasonably clickable if they were, so a hyperlink should be avoided in this case.

```suggestion
        util: 'SSEEdit v4.0.4'
```

File: masterlist.yaml [hunk 2]
```diff
@@ -15236,11 +15236,19 @@ plugins:
...
-        crc: 0x35A61832
-        util: '[SSEEdit v4.0.3](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
-        itm: 6916
+        crc: 0x010D8C29
+        util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        itm: 7
+  - name: 'MILightsControl.esp'
+    clean:
+      - crc: 0xC3C4CE49
+        util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
```
Comment (sibir-ine) [inline masterlist.yaml]: See comment above.

```suggestion
        util: 'SSEEdit v4.0.4'
```

**Review by sibir-ine — DISMISSED** [review]
@IllusiveMan196 Just requires a few changes.

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
@IllusiveMan196 Thanks.

**Pull request merged**